LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -182,6 +180,14 @@ public protocol PumpManager: DeviceManager {
     ///   - completion: A closure called after the command is complete
     ///   - result: A BasalRateSchedule or an error describing why the command failed
     func syncBasalRateSchedule(items scheduleItems: [RepeatingScheduleValue<Double>], completion: @escaping (_ result: Result<BasalRateSchedule, Error>) -> Void)
+
+    /// Sync the delivery limits for basal rate and bolus. If the pump does not support setting max bolus or max basal rates, the completion should be called with success including the provided delivery limits.
+    ///
+    /// - Parameters:
+    ///   - deliveryLimits: The delivery limits
+    ///   - completion: A closure called after the command is complete
```

Comment:
I assume that if the pump ends up setting different rates than were passed in, the completion handler will return the actual set values?

Reply:
Yes, that is the idea. I'll update the comment.

Reply:
Oh... I see that this is included in the result comment. @ps2 is this a blocker for you?

Reply:
Thanks, this is fine.